packages/site/src/layout/NavMenu.module.css

```diff
@@ -109,3 +109,9 @@
   }
 }
 
+:root[data-theme="dark"] .navMenuContainer {
```

Comment:
We use the blend mode to allow for two discrete hover states
![image](https://github.com/user-attachments/assets/28e7842b-4ffa-4f10-b3de-605e206fc835)
![image](https://github.com/user-attachments/assets/7ff5e145-c81d-4ce8-afce-c734231cc88d)